iati/tests/test_default.py

```diff
@@ -88,18 +93,36 @@ def test_default_codelist_invalid_at_all_versions(self, name, standard_version_o
         assert 'There is no default Codelist in version' in str(excinfo.value)
 
     def test_default_codelists_type(self, codelist_lengths_by_version):
-        """Check that the default Codelists are of the correct type.
-
-        Todo:
-            Check internal values beyond the codelists being the correct type.
-        """
+        """Check that the default Codelists are of the correct type."""
         codelists = iati.default.codelists(codelist_lengths_by_version.version)
 
         assert isinstance(codelists, dict)
         assert len(codelists.values()) == codelist_lengths_by_version.expected_length
         for codelist in codelists.values():
             assert isinstance(codelist, iati.Codelist)
 
+    def test_default_codelists_code_values(self, standard_version_optional, codelists_with_no_name_codes):
```

Comment:
I'm not sure this test is checking the right things? If it's purpose is to check for relevant data, why is it only doing type checking and then asserting something is merely not an empty string. If you want to ensure it has the relevant data it makes more sense to check for an expected value, rather than the absence of an unexpected value?

Reply:
The type checking should have been in a different test to separate responsibilities and keep things slightly DRYer.

https://github.com/IATI/pyIATI/pull/214/commits/092d7061414a84dfc8557e1b6603110a93d0aa09

Reply:
The check that it's not an empty string is all that can be done to check that *something* has been identified. Have modified the docstrings to clarify this, and renamed a function to clarify the current scope.

https://github.com/IATI/pyIATI/pull/214/commits/57910484547a166496f5c0690921273b74da782f